Backends/CLX/port.lisp

```diff
@@ -214,33 +214,24 @@
                       :map (sheet-enabled-p sheet)))
 
 (defmethod %realize-mirror ((port clx-port) (sheet top-level-sheet-pane))
-  (let ((q (compose-space sheet))
-        (frame (pane-frame sheet)))
-    (let ((window (realize-mirror-aux
-                   port sheet
-                   :map nil
-                   :width (round-coordinate (space-requirement-width q))
-                   :height (round-coordinate (space-requirement-height q))))
-	  (name (frame-pretty-name frame)))
-      (setf (xlib:wm-hints window) (xlib:make-wm-hints :input :on))
-      (setf (xlib:wm-name window) name)
-      (xlib:change-property window
-                            :_NET_WM_NAME
-                            (babel:string-to-octets name :encoding :utf-8)
-                            :UTF8_STRING 8)
-      (setf (xlib:wm-icon-name window) name)
-      (xlib:change-property window
-                            :_NET_WM_ICON_NAME
-                            (babel:string-to-octets name :encoding :utf-8)
-                            :UTF8_STRING 8)
-      (xlib:set-wm-class
-       window
-       (string-downcase (frame-name frame))
-       (string-capitalize (frame-name frame)))
-      (setf (xlib:wm-protocols window) `(:wm_delete_window))
-      (xlib:change-property window
-                            :WM_CLIENT_LEADER (list (xlib:window-id window))
-                            :WINDOW 32))))
+  (let* ((q (compose-space sheet))
+         (window (realize-mirror-aux
+                  port sheet
+                  :map nil
+                  :width (round-coordinate (space-requirement-width q))
+                  :height (round-coordinate (space-requirement-height q))))
+         (name (pane-name sheet))
```

Comment:
name should be sheet's property. panes are "sheets belonging to frames". In this case pane-name should be a method which resolves to sheet-name (to make this name consistent), so in clim-core/panes.lisp protocol class pane should not change

(name :initarg :name :initform "(Unnamed Pane)" :reader pane-name)

but we could add a comment that this slot name is the same as in named-sheet-mixin, so in top-level-sheet-pane both methods would refer to it.